.github/workflows/build.yml

```diff
@@ -32,10 +32,9 @@ jobs:
         uses: docker/setup-qemu-action@v3
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
-      - uses: aws-actions/configure-aws-credentials@v4
-        with:
-          aws-region: eu-west-1
-          role-to-assume: ${{secrets.GU_ECR_ROLE_ARN}}
+              with:
+                aws-region: eu-west-1
+                role-to-assume: ${{secrets.GU_ECR_ROLE_ARN}}
       - uses: aws-actions/amazon-ecr-login@v2
```

Comment:
I don't think the syntax is correct here, you're getting an error: https://github.com/guardian/recipes-backend/actions/runs/18747257385

Also the link you posted is about the riff-raff action, but this workflow doesn't use the riff-raff action, I think the aws credentials are used for the ECR login